### ISSUE TITLE
example bug？

### ISSUE THREAD

**Issue by mhzdlm** [apache/incubator-mxnet#2271]
in exapmle page。
https://github.com/dmlc/mxnet/tree/master/example/image-classification

Use Your Own Datasets

has that code

train_iter = mx.io.NDArrayIter(train_data, train_label, batch_size=batch_size, shuffle=True)

val_iter = mx.io.NDArrayIter(val_data, val_label, batch_size=batch_size)

model.fit(X=train_data, y=train_label)

train_iter bug use train_data

is a bug？

**Comment by mhzdlm** [comment]
英语太烂，中文再来一遍。

example的例子页面，使用自己的数据时，
train_iter = mx.io.NDArrayIter(train_data, train_label, batch_size=batch_size, shuffle=True)

val_iter = mx.io.NDArrayIter(val_data, val_label, batch_size=batch_size)

model.fit(X=train_data, y=train_label)

构建了两个intel迭代器，然后fit的时候直接用train_data,根本没用到itel，应该是写错了吧？

**Comment by mhzdlm** [comment]
正确用法，应该是这样吧

model.fit(X = train_iter, eval_data = val_iter)

**Comment by piiswrong** [comment]
Yes it's a typo. Would you like to submit a PR to fix this?

**Comment by mhzdlm** [comment]
i just use github 5 days,so i doest know how to fix it.

but i can try.

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
